features/karaf/openhab-tp/src/main/feature/feature.xml

```diff
@@ -267,37 +249,25 @@
 		<bundle dependency="true">mvn:org.eclipse.xtext/org.eclipse.xtext.xbase.ide/2.19.0</bundle>
 	</feature>
 
-	<!-- <feature name="openhab.tp-swagger-annotations" description="Swagger Annotations" version="${project.version}"> <bundle dependency="true">mvn:io.swagger/swagger-annotations/1.5.5</bundle> </feature> -->
-
-	<feature name="openhab.tp-swagger-jax-rs-provider" description="JAX-RS provider for Swagger integration" version="${project.version}">
+	<feature name="openhab.tp-jaxrswb-swagger1-gen" description="JAX-RS Whiteboard Swagger 1 generator" version="${project.version}">
 		<details>
 			This feature is currently only for external usage by products and not used by openHAB Core itself.
 		</details>
-		<capability>openhab.tp;feature=jax-rs-provider-swagger;version=1.1.1</capability>
-		<feature dependency="true">openhab.tp-jax-rs-min</feature>
+		<capability>openhab.tp;feature=jaxrswb-swagger1-gen;version=0.0.4</capability>
+		<bundle>mvn:de.maggu2810.jaxrswb.bundles/jaxrswb-swagger1-gen/0.0.4</bundle>
+		<feature dependency="true">openhab.tp-jax-rs-whiteboard</feature>
 		<feature dependency="true">openhab.tp-jackson</feature>
-		<!-- BEG: swagger-jersey2-1.5.8 -->
-		<feature dependency="true">openhab.tp-kat.cpy-jersey-min-2.22.2</feature>
-		<bundle>mvn:de.maggu2810.thirdparty.modified.io.swagger/swagger-jersey2-jaxrs/1.5.8.v20160511-1038</bundle>
-		<bundle dependency="true">mvn:io.swagger/swagger-annotations/1.5.8</bundle>
-		<bundle dependency="true">mvn:io.swagger/swagger-core/1.5.8</bundle>
-		<bundle dependency="true">mvn:io.swagger/swagger-jaxrs/1.5.8</bundle>
-		<bundle dependency="true">mvn:io.swagger/swagger-models/1.5.8</bundle>
-		<bundle dependency="true">mvn:de.maggu2810.thirdparty.modified.org.reflections/reflections/0.9.10.v20160429-1435</bundle>
-		<bundle dependency="true">mvn:com.google.code.findbugs/annotations/2.0.1</bundle>
-		<bundle dependency="true">mvn:com.google.guava/guava/18.0</bundle>
+		<bundle dependency="true">mvn:io.swagger/swagger-jaxrs/1.6.1</bundle>
```

Comment:
Can we use variable `${swagger.version}`?

Reply:
This property doesn't exist for openhab-core because Swagger is only defined here for the OH distro.